backend/core/src/seeder/seeds/listings/listing-default-multiple-ami-and-percentages.ts

```diff
@@ -10,16 +10,16 @@ export class ListingDefaultMultipleAMIAndPercentages extends ListingDefaultSeed
 
     const unitTypeOneBdrm = await this.unitTypeRepository.findOneOrFail({ name: "oneBdrm" })
 
-    const alamedaJurisdiction = await this.jurisdictionRepository.findOneOrFail({
-      name: CountyCode.alameda,
+    const bayAreaJurisdiction = await this.jurisdictionRepository.findOneOrFail({
+      name: CountyCode.bay_area,
     })
     const amiChartOne = await this.amiChartRepository.findOneOrFail({
       name: "San Jose TCAC 2019",
```

Comment:
Is there a reason we're keeping the San Jose and Alameda AMI charts?  For testing?

Reply:
see above, out of scope for this CL